src/openforms/js/components/admin/form_design/registrations/objectsapi/fields/DocumentTypes.js

```diff
@@ -1,8 +1,9 @@
 import classNames from 'classnames';
 import {useField, useFormikContext} from 'formik';
 import PropTypes from 'prop-types';
-import {useContext, useEffect} from 'react';
+import React, {useContext, useEffect} from 'react';
```

Comment:
we use the automatic jsx runtime, so the `React` import is not necessary :) (it's also not wrong)

Reply:
Ah okay, check. I'm also not sure why this was added... 😅
But i'll remove it :)